jwt-bcp/draft-sheffer-oauth-jwt-bcp.md

```diff
@@ -191,19 +325,42 @@ may sign JWTs with different "typ" values:
 * "JWT+set+oidc-logout" for OID Connect Logout events.
 
 Specifications that define JWT applications SHOULD define a unique "typ" value.
+-->
+
+Each application of JWTs defines a profile specifying the required and optional JWT claims
+and the validation rules associated with them.
+To prevent substitution of JWTs from one context into another, a number of strategies may be employed:
+
+* Use different "aud" values for different uses of JWTs within the same application.
+A straightforward way of doing this, for instance, is to have the audience be the URL of the endpoint
+at which that kind of JWT is to be used for Web-based applications.
```

Comment:
But neither the RFC nor the current document mandates that "aud" is the exact URL of the recipients endpoint.